### PR TITLE
Unify FP Views across Gobal & Application

### PR DESCRIPTION
**Pull request by zanthrash**
This creates one view that can be used across both the Gobal and Application views of
Fast Properties.

This also removes the depencancy on the V1 of the modal wizard.

Clean up of all the old Fast Property code and TS Convesion will be the next steps.

### CODE REVIEW

File: app/scripts/modules/netflix/fastProperties/dataNav/fastPropertyFilterSearch.less [hunk 1]
```diff
@@ -7,12 +7,14 @@
...
+  position: relative;
   margin-left: 10px;
   margin-top: 6px;
   .dropdown-menu {
+    position: absolute;
     min-width: 206px;
-    top: 38px;
-    left:560px;
+    top: 35px;
+    left:65px;
```
Comment (icfantv) [inline app/scripts/modules/netflix/fastProperties/dataNav/fastPropertyFilterSearch.less]: this is a big delta in the numbers, just making sure it was right.

**Comment by icfantv** [comment]
just the one question and LGTM.

File: app/scripts/modules/netflix/fastProperties/view/fastProperties.controller.js [hunk 2]
```diff
@@ -131,42 +158,59 @@ module.exports = angular
...
+          groups[key] = sortBy(groups[key], (prop) => prop.key.toLowerCase());
         }
         return groups;
       },
-      property: (propertiesList) => _.groupBy(angular.copy(propertiesList), 'key')
+      property: (propertiesList) => groupBy(angular.copy(propertiesList), 'key')
     };
 
     vm.filterAndGroup = (propertyList) => {
+      if(!isArray(propertyList)) {
```
Comment (anotherchrisberry) [inline app/scripts/modules/netflix/fastProperties/view/fastProperties.controller.js]: does anything yell at space between `if` and `(`?

**Comment by anotherchrisberry** [comment]
nice

File: app/scripts/modules/netflix/fastProperties/dataNav/fastPropertyFilterSearch.less (hunk 1 above)
Comment (zanthrash) [inline app/scripts/modules/netflix/fastProperties/dataNav/fastPropertyFilterSearch.less]: Yes, I also switched the positioning to be absolute within a relative container.

Comment (icfantv) [inline app/scripts/modules/netflix/fastProperties/dataNav/fastPropertyFilterSearch.less]: got it.

File: app/scripts/modules/netflix/fastProperties/view/fastProperties.controller.js (hunk 2 above)
Comment (zanthrash) [inline app/scripts/modules/netflix/fastProperties/view/fastProperties.controller.js]: No, It doesn't yell anymore.  Wierd.  I'll fix.

**Pull request merged**